### PR TITLE
IGAPP-1112: Parent breadcrumb cut

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

Open in mobile mode
http://localhost:9000/stadtdachau/de/allgemeine-beratungsstellen-und-beh%C3%B6rden/behoerden/jobcenter-dachau
Now all last breadcrumb items will not be cut until they reach 20 chars

### CODE REVIEW

**Review by charludo — APPROVED** [review]
works nicely, also looks like a very clean solution!

File: web/src/components/Breadcrumb.tsx [hunk 1]
```diff
@@ -31,13 +32,14 @@ const Separator = styled.span`
 
 type PropsType = {
   children: ReactNode
+  shrink?: boolean
```
Comment (steffenkleinle) [inline web/src/components/Breadcrumb.tsx:35]:
```suggestion
  shrink: boolean
```

File: web/src/components/Breadcrumb.tsx [hunk 2]
```diff
@@ -31,13 +32,14 @@ const Separator = styled.span`
...
 }
 
 /**
  * Displays breadcrumbs (Links) for lower category levels
  */
-const Breadcrumb = ({ children }: PropsType): ReactElement => (
-  <ListItem>
+const Breadcrumb = ({ children, shrink = false }: PropsType): ReactElement => (
```
Comment (steffenkleinle) [inline web/src/components/Breadcrumb.tsx:41]:
```suggestion
const Breadcrumb = ({ children, shrink }: PropsType): ReactElement => (
```

File: web/src/components/Breadcrumbs.tsx [hunk 3]
```diff
@@ -55,6 +55,8 @@ type PropsType = {
 const Breadcrumbs = ({ direction, ancestorBreadcrumbs, currentBreadcrumb }: PropsType): ReactElement => {
   // The current page should not be listed in the UI, but should be within the JsonLd.
   const jsonLdBreadcrumbs = [...ancestorBreadcrumbs, currentBreadcrumb]
+  // Min amount of chars when the last breadcrumb item should shrink
```
Comment (steffenkleinle) [inline web/src/components/Breadcrumbs.tsx:58]: Optional: Not sure if a comment is necessary here

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox, well done!

**Review by SteffiStuffel — APPROVED** [review]

File: web/src/components/Breadcrumbs.tsx (hunk 3 above)
Comment (LeandraH) [inline web/src/components/Breadcrumbs.tsx:58]: I think `Min text length after which the last breadcrumb item should shrink` is more understandable.

**Pull request merged**